reliability-v2/tasks/Tasks.py

```diff
@@ -0,0 +1,245 @@
+        self.logger.info(f"[Task] User {user}: delete project '{namespace}'")
+        (result,rc) = oc(f"delete project {namespace}",kubeconfig)
+        self.__log_result(rc)
+        return (result,rc)
+
+    # pod tasks in a namespace
+    def check_pods(self,user,namespace):
+        kubeconfig = global_data.kubeconfigs[user]
+        self.logger.info(f"[Task] User {user}: check pods in namespace {namespace}")
+        (result, rc) = oc(f"get pods -o wide -n {namespace}",kubeconfig)
+        #| egrep -v 'Running|Complete'
+        self.__log_result(rc)
+        return (result, rc)
+
+    # new an app in a namespace
+    def new_app(self,user,namespace):
+        kubeconfig = global_data.kubeconfigs[user]
+        template = random.choice(global_data.config["appTemplates"])["template"]
+        self.logger.info(f"[Task] User {user}: new app with tempate {template}")
+        (result,rc) = oc(f"new-app -n {namespace} --template {template}",kubeconfig)
+        if rc != 0:
+            self.__log_result(rc)
+            return(result,rc)
+        else:
+            (route,rc) = oc(f"get route --no-headers -n {namespace} | awk {{'print $2'}} | grep {template}",kubeconfig)
+            if rc == 0:
+                route = route.rstrip()
+                max_tries = 60
+                current_tries = 0
+                visit_success = False 
+                while not visit_success and current_tries <= max_tries:
+                    self.logger.info(f"{template} route not available yet, sleeping 10 seconds") 
+                    sleep(20)
+                    current_tries += 1
+                    visit_success = self.__visit_app(route)
+                if not visit_success:
+                    self.logger.error(f"add_app: {namespace}.{template} did not become available" )
+            self.__log_result(rc)
+            return(visit_success,0)
+
+    def __visit_app(self,route):
+        visit_success = False
+        try:
+            r = requests.get(f"http://{route}/")
+            self.logger.info(f"Response code:{str(r.status_code)}. Visit: {route}")
+            if r.status_code == 200:
+                visit_success = True
+        except Exception as e :
+            self.logger.error(f"Visit: {route}. Exception: {e}")
+        return visit_success
+
+    def load_app(self,user,namespace,clients="1"):
+        kubeconfig = global_data.kubeconfigs[user]
+        (route,rc) = oc(f"get route --no-headers -n {namespace} | awk {{'print $2'}}",kubeconfig)
+        route = route.rstrip()
+        self.logger.info(f"[Task] load app route {route} in namespace {namespace} with {clients} clients.")
+        urls = []
+        for i in range(int(clients)):
+            urls.append(route)
+        with ThreadPoolExecutor() as executor:
+            results = executor.map(self.__visit_app, urls)
+            return_value = 0
+            for result in results:
+                self.logger.info(result)
```

Comment:
This logging is causing a bit of log spam.  I suggest adding some more context or moving it to debug.   There are a few similar self.logger.info(result) instances as well.

2021-11-30 20:26:09,233 - INFO - True                                                                                                                                                                                                                                                     
2021-11-30 20:26:09,233 - INFO - True                                                                                                                                                                                                                                                     
2021-11-30 20:26:09,234 - INFO - True                                                                                                                                                                                                                                                     
2021-11-30 20:26:09,234 - INFO - True                                                                                                                                                                                                                                                     
2021-11-30 20:26:09,234 - INFO - True

Reply:
@mffiedler I also noticed this yesterday in my test. Will remove this log line.